client/my-sites/controller.js

```diff
@@ -271,9 +265,6 @@ module.exports = {
 		 * Sites is rendered on #primary but it doesn't expect a sidebar to exist
 		 * so section needs to be set explicitly and #secondary cleaned up
 		 */
-		context.store.dispatch( uiActions.setSection( 'sites', {
```

Comment:
I believe losing this dispatch is the cause of a few visual regressions:

Misalignment of My Sites selector: https://wordpress.com/stats

![Misalignment](https://cloud.githubusercontent.com/assets/1779930/13814231/dfc810b4-eb5b-11e5-8948-2e10d51b32ac.png)

White background in editor selector: https://wordpress.com/post

![White background](https://cloud.githubusercontent.com/assets/1779930/13814232/dfd75114-eb5b-11e5-9aef-74733363da97.png)